# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/VRParent.cpp
# Commit: f92f2cc29cb1
# Full Hash: f92f2cc29cb1e91274550f642affab21783ecfd5
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 2
# Description:
#   Bug 1550422 - P9. Sync preferences in VR process when they change. r=daoshengmu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31017
# ==============================================================================

diff -r 6dc82f88333d -r f92f2cc29cb1 gfx/vr/ipc/VRParent.cpp
--- a/gfx/vr/ipc/VRParent.cpp	Thu May 23 04:13:07 2019 +0000
+++ b/gfx/vr/ipc/VRParent.cpp	Thu May 23 04:13:07 2019 +0000
@@ -85,6 +85,11 @@
   return IPC_OK();
 }
 
+mozilla::ipc::IPCResult VRParent::RecvPreferenceUpdate(const Pref& aPref) {
+  Preferences::SetPreference(aPref);
+  return IPC_OK();
+}
+
 mozilla::ipc::IPCResult VRParent::RecvOpenVRControllerActionPathToVR(
     const nsCString& aPath) {
   mOpenVRControllerAction = aPath;